ubuntu/README.md

```diff
@@ -4,12 +4,20 @@ Ubuntu Docker in Docker image for gitlab-ci runners.
 
 
 ## How to use
+- Use dind image:
+Tagging convention: `docker:<docker_version>-dind`, or `docker:dind` for latest version.
```

Comment:
teracy/docker:<docker_version>-dind or teracy/docker:dind for **the** latest version.